projects/fal/src/fal/fal_script.py

```diff
@@ -13,7 +13,12 @@
 from dbt.config.runtime import RuntimeConfig
 from faldbt.logger import LOGGER
 
-from dbt.contracts.graph.parsed import ColumnInfo
+import faldbt.version as version
+
+if version.is_version_plus("1.4.0"):
+    from dbt.contracts.graph.nodes import ColumnInfo
+else:
+    from dbt.contracts.graph.parsed import ColumnInfo
```

Comment:
Aren't we dropping <1.4?

Reply:
Yes, but 

> - Left version-sensitive code in case we get requests and have to figure out a way to support more versions
>   - Probably could make it work for non-bigquery adapters

Reply:
Right!